lib/browser/api/web-contents.js

```diff
@@ -97,11 +98,20 @@ Object.setPrototypeOf(WebContents.prototype, NavigationController.prototype)
 // WebContents::sendToAll(channel, args..)
 WebContents.prototype.send = function (channel, ...args) {
   if (channel == null) throw new Error('Missing required channel argument')
-  return this._send(false, channel, args)
+  return this._send(false, false, channel, args)
 }
 WebContents.prototype.sendToAll = function (channel, ...args) {
   if (channel == null) throw new Error('Missing required channel argument')
-  return this._send(true, channel, args)
+  return this._send(false, true, channel, args)
+}
+
+WebContents.prototype._sendInternal = function (channel, ...args) {
+  if (channel == null) throw new Error('Missing required channel argument')
```

Comment:
Maybe we should check the first arg type instead of comparing it with `null` and `undefined`?

```js
if (typeof channel == 'string')
```

Reply:
@alexeykuzmin done